cypress-tests/cypress/support/commands/general.ts

```diff
@@ -2,8 +2,16 @@ import { TIMEOUTS } from "../../../config";
 
 export const validateLogin = {
   validate() {
+    cy.wait(10000);
```

Comment:
Waiting for a specific time [is strongly discouraged by the guidelines](https://docs.cypress.io/api/commands/wait#Time) -- I'll look into turning on the linter in these tests during the cooldown.

Unless it's a very short time span, like <=1s, it would be much better to wait for something to show up, or a response from an API, or any other event to happen.
Could you give a look into that? If it takes too long, please add a comment in the code about why we need to wait 10s here.

Reply:
Lorenzo we already have waits in other places... I.e. to wait for metadata indexing or when we call visitAndLoadProject to wait for things to render fully. 

I did not want to add more but there is no way I could think of to address this. The problem is that hitting the user endpoint on gitlab (via the gateway) right after completing the log in sometimes resulted in a 401 and sometimes in 200 (as expected). I think the problem is that our tests are just too fast for Gitlab or Gitlab is getting overwhelmed and fails to recognize or issue the tokens. It could also be the gateway but I looked through the logs of failed tests and did not see any errors or issues. Adding the wait completely removed this problem and Gitlab more consistently was able to log the user in and recognize the credentials.